### PR TITLE
feat(cli): Add --web option to use `mitmweb` instead of `mitmdump`

### LINKED ISSUE #10: Allow `mitmweb` UI instead of `mitmdump`
`mitmdump` is really nice, but the output with `roku --proxy` is currently very chatty. It'd be neat to at least have the MITM proxy UI available so that interactive changes can be made!

Let's consider a `--proxy-ui` argument that defaults to `dump` or something.  Or perhaps just `--use-mitmweb`? 🤷‍♂️

### PR DESCRIPTION
**Pull request by sjbarag**
The `mitmweb` interface is much more useful for interactive debugging than the `mitmdump` version, so let's make it available easily!

closes #10

### CODE REVIEW

File: roku_dev_cli/roku_dev_cli.py [hunk 1]
```diff
@@ -592,6 +594,7 @@ def main():
     parser.add_argument('-a', '--automation', action='store_true', help="creates an automation build")
     parser.add_argument('-p', '--proxy', action='store_true', help="builds the app to proxy through this host")
     parser.add_argument('-s', '--proxy_scripts', nargs='+', help='Provide paths to proxy add-on scripts', required=False)
+    parser.add_argument('-w', '--web', action='store_true', help='Uses mitmweb to provide a browser-based network monitor', required=False, default=None)
```
Comment (strattonbrazil) [inline roku_dev_cli/roku_dev_cli.py]: Nit, but why not default to False?

**Review by strattonbrazil — APPROVED** [review]
This looks like a good change.  I vaguely recall there was some nuance to scripts in mitmweb, but can't remember any examples.  Please test an example script to make sure it works.

File: roku_dev_cli/roku_dev_cli.py (hunk 1 above)
Comment (sjbarag) [reply roku_dev_cli/roku_dev_cli.py]: Whoops!  I'm used to the behavior of node's `yargs` or `commander` which automatically set up a negated command (i.e. `--no-web` in this case) and wanted to be able to detect either `--web` or `--no-web` without `--proxy`.  argparse doesn't do that, so I can default to `False`!

**Comment by sjbarag** [comment]
> This looks like a good change. I vaguely recall there was some nuance to scripts in mitmweb, but can't remember any examples. Please test an example script to make sure it works.

Good point!  I just tested a very simple outside of this CLI to test `mitmweb`: 

```python3
class ExampleError:
    def response(self, flow):
        request = flow.request
        response = flow.response

        if "example.com" in request.host_header:
            response.status_code = 500
            response.text = "mitmweb sent me"

addons = [ExampleError()]
```

Run `mitmweb -s /path/to/example-error.py`, configure Firefox to use `localhost:8080` as its proxy, and load `example.com` (you may need a cache-busting refresh).  You'll receive a 500 error and the `mitmweb sent me` message!

**Pull request merged**